dev/com.ibm.ws.ejbcontainer.session.passivation_fat/fat/src/com/ibm/ws/ejbcontainer/session/passivation/tests/StatefulTimeoutTest.java

```diff
@@ -48,7 +49,7 @@ public LibertyServer getServer() {
     }
 
     @ClassRule
-    public static RepeatTests r = RepeatTests.with(FeatureReplacementAction.EE7_FEATURES().forServers("com.ibm.ws.ejbcontainer.session.passivation.fat.sfTimeout")).andWith(FeatureReplacementAction.EE8_FEATURES().forServers("com.ibm.ws.ejbcontainer.session.passivation.fat.sfTimeout"));
+    public static RepeatTests r = RepeatTests.with(FeatureReplacementAction.EE7_FEATURES().forServers("com.ibm.ws.ejbcontainer.session.passivation.fat.sfTimeout")).andWith(FeatureReplacementAction.EE8_FEATURES().forServers("com.ibm.ws.ejbcontainer.session.passivation.fat.sfTimeout")).andWith(new JakartaEE9Action().forServers("com.ibm.ws.ejbcontainer.session.passivation.fat.sfTimeout"));
```

Comment:
I created an convenience method for this that you might wish to try...

https://github.com/OpenLiberty/open-liberty/blob/654502ab95d074975df65dd78baf9e3e0508d247/dev/fattest.simplicity/src/componenttest/rules/repeater/EERepeatTests.java#L61

Reply:
@tevans78 That is a nice convenience method, but I generally try to adhere to the FAT guidelines that indicate FAT should only run one repeat action in lite mode, the others in full mode.  I had overlooked that this test was repeating both 7 and 8 in lite mode already..... I've updated the test to repeat ee7 in full mode only.  For now, it will run both ee8 and ee9 in lite mode, as I don't want to lose the ee8 since it is the ga version, but would like to get some minimal ee9 testing.

Perhaps that convenience method could be updated to allow passing in something like ("myserver", EE7-FF, EE8, EE9-FF), to allow designating some to repeat full fat only?  We like to switch things up a bit, always have some FAT run lite mode for  EE7, some EE8, and eventually some EE9, but the majority run the latest ga in lite.